.github/workflows/publish.yml

```diff
@@ -0,0 +1,67 @@
+name: Publish Python distribution to PyPI
+
+on: push
```

Comment:
@NumanIjaz This action is triggered [on every push](https://github.com/NumanIjaz/steuer-id/actions/workflows/publish.yml) and build even if not publishable. I would rather use:

```
on:  
  push:  
    tags
```

to trigger only when new tags created.